python/demo/demo_navier-stokes.py

```diff
@@ -325,13 +325,20 @@ def jump(phi, n):
 bc_u = fem.dirichletbc(u_D, boundary_vel_dofs)
 bcs = [bc_u]
 
+
 # Assemble Stokes problem
-A = assemble_matrix_block(a_blocked, bcs=bcs)
+A = assemble_matrix(a_blocked, bcs=bcs)
 A.assemble()
-b = assemble_vector_block(L_blocked, a_blocked, bcs=bcs)
+
+b = assemble_vector(L_blocked, kind=PETSc.Vec.Type.MPI)
+bcs1 = fem.bcs_by_block(fem.extract_function_spaces(a_blocked, 1), bcs)
```

Comment:
Do you want to keep it in this way for coverage of the API?

Reply:
Yes